app/code/Magento/InventoryApi/Test/Acceptance/Data/MsiSourceData.xml

```diff
@@ -75,4 +75,14 @@
         <data key="postcode">QC G2E 6J5</data>
         <data key="use_default_carrier_config">1</data>
     </entity>
+    <!-- Please don't use this entity in your tests -->
```

Comment:
Please describe why this entity shouldn't be used in any other tests.

Reply:
Done